app/institutions/dashboard/-components/total-count-kpi-wrapper/component-test.ts

```diff
@@ -64,17 +69,62 @@ module('Integration | institutions | dashboard | -components | total-count-kpi-w
             .hasAttribute('data-icon', 'archive');
 
         // And the fourth total kpi is tested
-        assert.dom('[data-test-total-count-kpi="3"]')
+        let parentContainer = '[data-test-total-count-kpi="3"]';
+        assert.dom(parentContainer)
             .exists('The Preprint Widget exists');
 
-        assert.dom('[data-test-total-count-kpi="3"]')
+        assert.dom(parentContainer)
             .hasText('1000 OSF Preprints');
 
-        assert.dom('[data-test-total-count-kpi="3"] [data-test-kpi-icon]')
+        assert.dom(`${parentContainer} [data-test-kpi-icon]`)
             .hasAttribute('data-icon', 'file-alt');
 
-        // Finally there are only 4 widgets
-        assert.dom('[data-test-total-count-kpi="4"]')
-            .doesNotExist('There are only 4 widgets');
+        // And the total storage kpi is tested
+        parentContainer = '[data-test-total-count-kpi="4"]';
+        assert.dom(parentContainer)
+            .exists('The Total Storage Widget exists');
+
+        assert.dom(parentContainer)
+            .hasText('104593230 Total Storage');
```

Comment:
There's a `human-file-size` util that could be appropriate to use here. This will convert the number to KB, MB, GB, etc. We could add a getter (something like `get convertedStorageCount`?) to the institution-summary-metric model that takes the `storageByteCount` and returns the `human-file-size` version of that?